Mathlib/Analysis/Normed/Ring/Seminorm.lean

```diff
@@ -388,3 +388,52 @@ lemma MulRingNorm_nat_le_nat {R : Type*} [Ring R] (n : ℕ) (f : MulRingNorm R)
       f (n + 1) ≤ f (n) + f 1 := f.add_le' ↑n 1
       _ = f (n) + 1 := by rw [map_one]
       _ ≤ n + 1 := add_le_add_right hn 1
+
+noncomputable section
+variable {R : Type*} [Ring R]
```

Comment:
Where is this variable used?

Reply:
You're right, we never use it here. But seeing the next suggestion we can use it :D

Reply:
Since it is only used in that lemma, I would put it as an argument there instead of as a variable.

Reply:
That makes sense!